services/opencollective/opencollective-base.js

```diff
@@ -38,45 +42,49 @@ export default class OpencollectiveBase extends BaseJsonService {
     }
   }
 
-  async fetchCollectiveInfo(collective) {
-    return this._requestJson({
-      schema: collectiveDetailsSchema,
-      // https://developer.opencollective.com/#/api/collectives?id=get-info
-      url: `https://opencollective.com/${collective}.json`,
-      httpErrors: {
+  async fetchCollectiveInfo({ collective, accountType }) {
+    return this._requestGraphql({
+      schema,
+      url: 'https://api.opencollective.com/graphql/v2',
+      query: gql`
+        query account($slug: String, $accountType: [AccountType]) {
+          account(slug: $slug) {
+            name
+            slug
+            members(accountType: $accountType, role: BACKER) {
+              totalCount
+              nodes {
+                tier {
+                  legacyId
+                  name
+                }
+              }
+            }
+          }
+        }
+      `,
+      variables: {
+        slug: collective,
+        accountType,
+      },
+      httpErrorMessages: {
         404: 'collective not found',
       },
+      options: {
+        headers: { 'content-type': 'application/json' },
+      },
     })
   }
 
-  async fetchCollectiveBackersCount(collective, { userType, tierId }) {
-    const schema = buildMembersArraySchema({
-      userType:
-        userType === 'users'
-          ? 'USER'
-          : userType === 'organizations'
-          ? 'ORGANIZATION'
-          : undefined,
-      tierRequired: tierId,
-    })
-    const members = await this._requestJson({
-      schema,
-      // https://developer.opencollective.com/#/api/collectives?id=get-members
-      // https://developer.opencollective.com/#/api/collectives?id=get-members-per-tier
-      url: `https://opencollective.com/${collective}/members/${
-        userType || 'all'
-      }.json${tierId ? `?TierId=${tierId}` : ''}`,
-      httpErrors: {
-        404: 'collective not found',
+  getCnt(data) {
+    const {
+      data: {
+        account: {
+          members: { totalCount: cnt },
```

Comment:
Can we just use `count`/`getCount()`. Count is not a long word and readability matters. Lots of our contributors are not first language English speakers.